<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContractBalanceWidget</class>
 <widget class="QWidget" name="ContractBalanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="depositBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>85</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>充值合约</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="balancesTableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>118</y>
     <width>650</width>
     <height>294</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>币种</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>余额</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>28</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>合约余额</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>65</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: rgb(80,80,80);
font: 11px &quot;微软雅黑&quot;;}</string>
   </property>
   <property name="text">
    <string>合约地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="contractAddressLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>85</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: rgb(80,80,80);
font: 11px &quot;微软雅黑&quot;;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QToolButton" name="openForUsersBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>85</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>开放合约</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>448</y>
     <width>770</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
